client/containers/Budget/index.js

```diff
@@ -25,9 +25,9 @@ class Budget extends Component {
     const data = { transactions, categories };
 
     return (
-      <div>
-        <BudgetGrid data={data} />
-      </div>
+      <Chunk load={() => import('components/BudgetGrid')}>
```

Comment:
My editor's linter hates the import statement here. There must be an eslint setting to make it be quiet. Not sure if you used one in your local or not or if you know of one offhand?